src/MTGCore/Services/Exceptions/SearchFilterMapperException.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace MTGCore.Services.Exceptions
+{
+    public class SearchFilterMapperException : Exception
+    {
+        public SearchFilterMapperException(string message) : base(message)
+        {
```

Comment:
I probably wouldn't bother implementing a custom exception unless you plan on using it. (Maybe a try catch could be useful somewhere)